### ISSUE TITLE
Proposed payment method addition process and estimated  timelines

### ISSUE THREAD

**Issue by pazza83** [bisq-network/growth#206]
### Proposed payment method addition process and estimated timelines

Part of the [Payment Method Maintainer](https://github.com/bisq-network/roles/issues/38) role is to develop a process for payment methods to be added.

I have documented my proposal below for discussion / comments.

As the role requires input from a numerous Bisq Teams I keen to ensure quick progress though all stages, therefore, I have also proposed estimated timescales. I am keen to speed up what appears occasionally to have been a prolonged process.

In order to achieve the proposed timelines for multiple concurrent payment methods. I am keen to avoid discussions that require significant changes to existing trade protocols.  While such discussions and proposals are useful to the development of Bisq I feel these discussions fall outside the role of Payment Method Maintainer and are best discussed as separate issues. Payments that have little to no change in existing trade protocols needed can be added quickly and will keep resources required to a minimum.

The below proposal has just been created from my current understanding of the role and is therefore, a work in progress, I welcome any feedback.

***

**0-60 days**
---
1. User requests payment method on Bisq channel; GitHub, Twitter, Reddit, Telegram, KeyBase, Bisq Community etc  

2. User directed to create issue on [Bisq-Network>Growth>Issues](https://github.com/bisq-network/growth/issues)  using the '[Issue: New payment method](https://github.com/bisq-network/growth/issues/new?assignees=&labels=re%3Apayment-method&template=add_payment_method.md&title=)' template, or in some instances Payment Method Maintainer will create it on their behalf.

3. Issue discussed by all interested stakeholders with regards to if payment methods meets the ‘[requirements of a payment method](https://github.com/bisq-network/bisq/issues/4757#issuecomment-727711700)’ to be added to Bisq.

4. Payment Method Maintainer recommends one of the following:
        
        a. Payment method not to be added to Bisq.
        b. Payment method to be added to Bisq.

**61-90 days**
---
5.	If Payment Method Maintainer recommends against adding the payment method then user can submit proposal to the DAO for a vote.  

6.	If Payment Method Maintainer recommends payment method to be added to Bisq they will@bisq-network/mediators and @bisq-network/arbitrators informing them of the intention to add the new payment method along with the proposed requirements; limits, aging, signing requirement, required fields, etc.
        
        a.	If no objections payment method will be prioritized for addition.

7.	If @bisq-network/mediators or @bisq-network/arbitrators have concerns these can be discussed. 
        
        a.	If rough consensus reached payment method will be prioritized for addition.
        b.	If rough consensus not reached payment method, Payment Method Maintainer, will put the issue to DAO vote.  

**91-150 days**
---

8.	If Payment Method Maintainer recommends against a payment method to be added, or if the DAO vote against adding a payment method, the payment method will not be considered for addition for at least 12 months. This will be documented on the payment methods roadmap.

9.	If Payment Method Maintainer recommends a payment method to be added, or if the DAO vote for adding a payment method, the payment method will be prioritized for addition and added to the roadmap. The Payment Method Maintainer will then:

        a.	Package the required information to make it easy for @bisq-network/bisq-devs to add the payment method
        b.	Update Bisq Wiki to include payment method information.

**0-150 days from payment method being added**
---
10.	Once new payment method added Payment Method Maintainer will liaise with @bisq-network/growth to promote the new payment method on various channels; website, wiki, twitter, reddit, telegram, bisq community etc.
  
11.	Once the new payment method is launched Payment Method Maintainer will assist with the onboarding and and snagging required during its first 6 months with a view to maximize it’s adoption, or, in adverse situations it’s retirement.

12.	Payment Method Maintainer will then report on status, KPIs, metrics of any new payment methods added.

**Comment by chimp1984** [comment]
The point with promoting a method and to verify afterwards its success are very important points and I think lack of promotion was one of the reasons some did not get more traction.
Would be great it we can get this done with the 2 new ones for 1.5.0.

Also I think the experience with CBM was/is a bit an extreme case. I don't remember other payment methods which caused that much of discussions and different opinions. F2F had similar situation but was less contentious. So just wanted to mention that so that you not assume that each new method will cause so much discussion effort. ;-)

**Comment by pazza83** [comment]
Hi @chimp1984  

Many thanks for your comments. 

Yes, I am going to look into getting some graphics made for socials and also adding info about it on the Wiki. 

Was musing about the possibility of having 0% maker fees for first 3 months or so with new payment methods. That way it would get offers up for them on the client that traders would see and increase awareness. Plus Bisq would still get the more lucrative taker fees.

Not sure if maker fees can be set per payment method in the client or not? 

Haha, I don't mind. Rather have the contentious ones to start with :)

**Comment by chimp1984** [comment]
>  0% maker fees

I fear that would cause too much extra technical effect as the trade protocol expects a maker fee tx.

> Not sure if maker fees can be set per payment method in the client or not?

Not yet, but that would be less of an effort.

**Comment by pazza83** [comment]
Noted, likewise, other platforms are charging 1% maker fees for CBM trades. Could be a way to increase revenue for Bisq for payment methods with more mediation complexities.

**Comment by ripcurlx** [comment]
I think it is good to have a timeline for new payment methods for expectation management. The main point as mentioned above is to have someone that does the bootstrapping for a new payment method putting out offers to be taken. Additionally I think we should advertise (twitter,...) interesting new payment methods already before they get implemented to gain interest in the community. Also to make sure the implementation part of the payment method is done in time it would be great to have a dedicated `payment method dev`. It is not very complicated task to do so I think also a new developer could do this just fine. If you have done it once and all translations and specifications are ready it is a very quick process to add a new payment method.

**Comment by pazza83** [comment]
Thanks, going to have a think of ways to effectively promote a new payment method when added to allow it to become established.

A dedicated payment method dev would be great.  I could also try and batch up new payment methods to be added to reduce dev time.

**Comment by m52go** [comment]
This looks good. As for recent promotion efforts, we ran a Twitter campaign on US Postal Money Orders that did fairly well. Volume went up notably afterward and some of the initial spike seemed to stick. We also created [graphics for USD and EUR](https://bisq.wiki/Payment_methods#Payment_method_guides) to advise on pros and cons of the major payment methods. Maybe similar graphics could be created that are focused on specific new payment methods. 

As mentioned above, seeding the market with offers would be important. This was the impetus for the USPMO campaign -- someone did that, which made promotion feasible.

**Comment by pazza83** [comment]
Thanks that is useful I will look to create some graphics for socials for the new payment methods to be added. Would be good if they could also link to a campaign. I agree no point advertising that they can buy in XXX if there are no XXX for them to take.  If there any way to incentivize people to place maker offers with a new payment method?

**Comment by m52go** [comment]
Bounties have been tried to bootstrap new markets, but could be tried for new payment methods in less-active markets.

If market is already active, it might be sufficient to just market the new method well and find 1-2 people to create offers.

Would be nice to think of other ways...but privacy limitations make it difficult to reward user activity, so need to be clever. Maybe reaching out to traders already using the payment method on other platforms is one way.

**Comment by pazza83** [comment]
That is great, thanks for all the info.

**Comment by chimp1984** [comment]
@m52go Have bounties show real sustained success? E.g. did we ever manage to bootstrap a new market mainly based on bounties?
I am asking as I made repeatedly the experience that money is the wrong motivator for Bisq.

**Comment by m52go** [comment]
I don't personally like bounties. In my limited experience people do what's required of the bounty and then seem to stop working. Could be bad bounty structure but probably more like what you said -- money is wrong motivator.

I just suggested them as something tried before, but would much prefer to find/use other ways.

**Comment by dave2014btc** [comment]
https://github.com/bisq-network/bisq/issues/1101#issuecomment-731097699 - my two cents not worth copying as its more for the other convo but has my ideas on it. also 1% fee is fair i dont mind paying that on my other websites.

**Comment by dave2014btc** [comment]
you will not need to do anything to promote cash in mail, they are enough big trades with lots of customers on other websites which will move over soon as they can, they are my competition and every new site that allows CIM they as on there within a week. chances are they will move customers over as no one likes the idea of centralised escrows.

**Comment by pazza83** [comment]
> 1% fee is fair i dont mind paying that on my other websites.

Thanks for the feedback. That is good to know.

**Comment by m52go** [comment]
@dave2014btc do you know of a good way to reach offer makers on other websites? Are there forums or chat groups where they tend to hang out? I noticed some have direct email addresses listed on their profiles but many don't.

**Comment by m52go** [comment]
Was just pondering ways to incentivize market makers. Instead of offering bounties for making offers, what if we offered refunds for maker fees as an incentive for special events like liquidity days and new market pushes (and possibly new payment methods)? Not all the time, just occasionally for special events.

I know maker fees on Bisq are relatively low, but LocalMonero and LocalBitcoins are _free_. You only pay after completing a deal. Bisq has added downside of complexity and hassle of keeping program running for offers to stay live.

If nothing else maybe it could be an attractive marketing point for traders to try Bisq. Would of course need to set appropriate terms so it's not abused, but overall the incentives seem to make more sense for both sides (traders encouraged to make enough offers to make market look attractive; growth team encouraged to do enough marketing to avoid paying refunds). In worst case, even a refund request for 0.5 BTC worth of offers would result in a maximum of 0.0005 BTC cost to the DAO, far less than past bounties.

Any thoughts?

**Comment by chimp1984** [comment]
I fear the administrative overhead and miner fees are too high. I also think we should try to focus on other incentives than money as that has never really worked in the past.

**Comment by crocket** [comment]
I'd also like to pay maker fee after completing a deal..

**Comment by m52go** [comment]
I agree about money as bad motivator for general contributors. The fees in question are almost negligible and trader would need to spend upfront anyway...there's no profit in it. My intention was more to explore ways to reduce friction that active traders don't get elsewhere as temporary carrots.

**Comment by m52go** [comment]
> I'd also like to pay maker fee after completing a deal..

That's not possible. Fee needs to be paid when offer is made, per the current trade protocol. I was just exploring the idea of refunding trade fees for new market makers in new markets if they fail to be taken after a certain amount of time as a temporary incentive.

**Comment by chimp1984** [comment]
> > I'd also like to pay maker fee after completing a deal..
> 
> That's not possible. Fee needs to be paid when offer is made, per the current trade protocol.

With the new trade protocol idea (1 tx protocol) we are more flexible with changes regarding fees, but now its not easy/possible to remove maker fee.

**Comment by pazza83** [comment]
> Was just pondering ways to incentivize market makers. Instead of offering bounties for making offers, what if we offered refunds for maker fees as an incentive for special events like liquidity days and new market pushes (and possibly new payment methods)? Not all the time, just occasionally for special events.

I do not think a refund of maker fees would be much incentive for me personally. They are generally small and it seems like to claim them would be more work than it is worth. Also conscious that someone would have to check and process them.  

Not sure what would be a good alternative. Maybe a reduction in maker fees for new payment methods, promotions etc. But this would likely take too much of the devs resources.

Generally, I think I would place more offers on Bisq if I could place multiple offers at once. Trying to place x10 0.01 BTC offers takes much longer than I would like.

**Comment by chimp1984** [comment]
> Generally, I think I would place more offers on Bisq if I could place multiple offers at once. Trying to place x10 0.01 BTC offers takes much longer than I would like.

The API should be ready soon for production. That will make it much easier to automate trades with scripts.

**Comment by pazza83** [comment]
That is great. I want to get them all for low miner fees. It is a bit of a lottery at the moment. Not places a trades since the new release yet so hoping that might might a difference.

Could the API be set up to place X amount of maker trades, on the XXX / BTC market, for X BTC, for X% over market rate, using X payment method, if miner fees are <X?

This would save hours!!

**Comment by chimp1984** [comment]
Yes thats will be an interesting use-case. Write a script which waits until miner fee is below a certain amount for placing offers. 
@ghubstan is the dev behind the API. Best to tell him what are your requirements so he can tailor the API for real traders needs. But basically the API is only offering the basic functionalities, the logic need to be done in some script which used the API. But would be good if @ghubstan can provide such sample scripts so it gets easier to traders to adjust/use them.

**Comment by m52go** [comment]
As a start maybe we can develop and implement publicity campaigns for payment methods like we do for markets.

For example [this upcoming liquidity drive](https://github.com/bisq-network/growth/issues/211) is currently planned around CAD, but maybe we can also target TransferWise and Amazon at the same time. 

Perhaps just 1-2 market makers recruited through existing Bisq channels (Keybase, GitHub, Telegram, etc) along with a solid lineup of videos and articles placed in the right channels (market-specific media) could be effective without any financial incentives.

**Comment by pazza83** [comment]
Yes, I think that would be great. I am going to open up an issue about ideas for marketing new payment methods. 

- Amazon eGift Cards has 2 live offers
- TransferWise has 16 live offers
- PayID has no live offers yet

I like the idea of recruiting market makers. I agree that market making should be profitable on it's own and therefore not  require financial incentives.

**Comment by Conza88** [comment]
Yes. Efforts towards highlighting the opportunity etc, new markets etc.

Also those looking for no KYC bitcoin, how does Bisq rank on searches?

Worth some paid articles in various crypto news outlets?
________________________________

**Comment by pazza83** [comment]
Yes , these sound good. 

Not sure about paid articles, unless there would be something that would be a really good fit.

There is a BSQ Bounty out for SEO Analysis https://github.com/bisq-network/growth/issues/200

**Comment by pazza83** [comment]
I will use the following template when recommending payment methods to be added.

This is to ensure mediators, arbitrators, developers and users are clear about how the payment method will fit within Bisq.

### Proposed National Currency Payment Method Template

---

Date proposal being made:
Date feedback to be provided by:

Name of payment method:

Required fields (eg)
- Region
- Country
- Account owner full name
- Email
- Phone number
- User name
- Bank name
- Account number
- Bank code
- Extra requirements

Maximum trade duration:

Signing required: Yes/No

Regions accepted

Currencies accepted

Bisq Limits: eg 0.25 BTC

Payment method limits: eg $1000 USD per payment 

Any relevant information for mediators / arbitrators:

Any relevant information for devs:

Any relevant information for users:

Screenshot of expected GUI:** 

![new account](https://user-images.githubusercontent.com/66367547/115619978-827ace80-a2ec-11eb-87d7-3da6fe1b2e6e.PNG)
